packages/api/prepare.go

```diff
@@ -132,7 +137,26 @@ func forsignFormData(w http.ResponseWriter, r *http.Request, logger *log.Entry,
 				val = r.FormValue(fitem.Name)
 				req.SetValue(fitem.Name, val)
 			}
-		} else {
+
+		case script.Decimal:
```

Comment:
I think it would be better to move that case little bit upper, with fitem.ContainsTag('money'), to make it little more obviouse, but it's at your decision